integration-tests/client-rpc/test/hdrestore-auto-sync.test.ts

```diff
@@ -160,16 +160,22 @@ describe("Wallet Auto-sync", () => {
 			},
 			"Sender should have one Outgoing transaction",
 		);
-
-		const senderWalletBalanceAfterSend = await asyncMiddleman(
+        // after sync, the pending balance will become available
+		const senderWalletBalanceAfterSync = await asyncMiddleman(
 			zeroFeeRpcClient.request("wallet_balance", [senderWalletRequest]),
 			"Error when retrieving sender wallet balance after send",
 		);
-		expect(senderWalletBalanceAfterSend).to.eq(
-			new BigNumber(senderWalletBalanceBeforeSend)
-				.minus(transferAmount)
-				.toString(10),
-			"Sender balance should be deducted by transfer amount",
+        const returnAmount = new BigNumber(senderWalletBalanceBeforeSend.total)
+            .minus(transferAmount)
+            .toString(10);
+        const expectBalanceAfterSync = {
+            total: returnAmount,
+            available: returnAmount,
+            pending: "0",
+        }
```

Comment:
the indentation seems to be inconsistent with other lines.

Reply:
I'll use a IDE formatter to fix it.

Reply:
fixed